### DISCUSSION TITLE
Readonly settings number fields still have stepper buttons

### DISCUSSION THREAD

**Discussion by jtbandes** [foxglove/community#552] [Q&A]
These settings are readonly (from a URDF in the 3D panel) but they still have buttons that make it seem like I can change them:

<img width="301" alt="image" src="https://user-images.githubusercontent.com/14237/200452159-626ba202-b1fa-4284-803c-881252c4c235.png">

**Comment by defunctzombie** [comment]
Could see this being a bug in the settings ui. Certainly if I was a panel author and this was the experience my users had I'd think it is a bug with the `readonly` property. At least you can't edit the property, _that_ would be a bug for sure :D
